portfolio/src/main/webapp/script.js

```diff
@@ -174,6 +174,21 @@ function deleteData() {
   });
 }
 
+/**
+ * Loads the blobstore image storage and shows the form to submit an image.
+ */
+function loadBlobstoreAndForm() {
+  fetch('/blobstore-upload-url')
+      .then((response) => {
+        return response.text();
+      })
+      .then((imageUploadUrl) => {
```

Comment:
Do you mean uploadUrl? imageUploadUrl isn't in this function or in the get function for blobstore-upload-url

Reply:
Since this is a .then function, I'm just naming the text that the promise returns imageUploadUrl. But it ends up being the same as uploadUrl, so I will rename it that way so it's more clear.